tests/profiler/test_profiler.py

```diff
@@ -303,50 +303,48 @@ def test_pytorch_profiler_trainer_ddp(tmpdir, pytorch_profiler):
         assert any(f"{local_rank}-validation_step" in f for f in files)
 
 
-def test_pytorch_profiler_trainer_test(tmpdir):
+@pytest.mark.parametrize("boring_model_cls", [BoringModel, ManualOptimBoringModel])
+def test_pytorch_profiler_trainer_fit(boring_model_cls, tmpdir):
```

Comment:
can the fit case not be covered by the test below? what could be the difference?

Reply:
In fit, there is 2 traces generated, one for train, one for val.

Reply:
It was more readable separated.